website/src/components/Documentation.tsx

```diff
@@ -123,7 +125,8 @@ export default function Documentation({
 									onClick={(evt) => {
 										evt.preventDefault()
 										evt.stopPropagation()
-										setCurrentTarget(pathToRules[to])
+										const { pathname, search } = evt.currentTarget
+										setCurrentTarget(pathToRules[decodeURI(pathname)], search)
```

Comment:
La gymnastique `pathToRules[decodeURI(pathname)]` est rendue nécessaire parce que le composant `<Link />` qu'un utilisateur peut fournir à la documentation reçoit uniquement une URL `to` et non le nom de la règle. Cela pourrait être changé j'imagine.

(dans la version d'avant on utilisait la prop `to`, qui permet de se passer de `decodeURI` mais qui implique de reparser la chaîne `ma/regle?subengine=1` pour extraire le chemin `ma/regle` et les paramètres GET)